docs/ops.md

```diff
@@ -61,21 +61,21 @@ If you are using systemd as your init system, which most modern linux OSs are, t
 
 ```systemd
 [Unit]
-Description=containerd container runtime
-Documentation=https://containerd.io
-After=network.target
+Description="containerd container runtime"
+Documentation="https://containerd.io"
+After="network.target"
```

Comment:
Do you have more information about the issue you ran into? Because these should all be valid.

Quoting won't be needed for most cases, except for case where special values (e.g. `=`,  `;`, `#`) or leading/trailing whitespace must be preserved.  https://man7.org/linux/man-pages/man7/systemd.syntax.7.html#QUOTING